include/exadg/solvers_and_preconditioners/multigrid/multigrid_preconditioner_base.cpp

```diff
@@ -306,6 +306,82 @@ MultigridPreconditionerBase<dim, Number>::check_levels(std::vector<MGLevelInfo>
   }
 }
 
+
+
+template<int dim, int spacedim>
+std::vector<std::shared_ptr<const Triangulation<dim, spacedim>>>
```

Comment:
sorry for the inconvenience, but in ExaDG we want to consistently use `const` as a trailing keyword ;)